source/development/feature-flags.html.md

```diff
@@ -0,0 +1,76 @@
+
+### The thing might be a long term project for users
+When assessing if a breaking change should be behind a feature flag, consider the scope of work required by the user. A feature flag is useful if the change is very difficult to implement and/or has wide scope. Using the green link example in the previous section, this would impact all links in a given service so that service will likely need to check all their pages to ensure the design of their pages still made sense against the new link colour.
+
+Feature flags are more useful for wide scope changes that are likely to take a long time to action over difficult to implement changes. For difficult to implement but short term changes, consider using a pre-release instead (see below section on pre-releases).
+
+### We are unsure of the thing’s value
+This is an alternative to the previous principal on implementation difficulty where, for whatever reason, we have a significant change that we aren’t 100% confident in and have been unable to verify its value. In this instance, putting it behind a feature flag would allow us to get early feedback from users who’re willing to use and test our new/updated feature and give us feedback, allowing us to iterate the change with data.
+
+## When a feature flag isn't useful
+Even if you have a scenario where feature flags may be useful, consider the following before implementing one.
+
+### When releasing the thing carries no significant risk
+We should avoid using feature flags in most cases. We have alternative systems for communicating big or impactful changes such as feature depreciation where we tell people we’ll be removing something in a future breaking release and breaking releases themselves where users know that by upgrading, they might need to change something in their service. Feature flags should be used as a way to alleviate instances where releasing as normal is challenging.
+
+### We could use a pre-release instead
+Another method we have for testing breaking changes before release are pre-releases. This is a version of govuk-frontend that we release to npm like normal releases, but we tag it as a “beta” release eg: instead of 5.0.0, we might release 5.0.0-beta.0. This communicates that it isn’t the next major version of govuk-frontend but an early, “experimental” version of that release, allowing users to assess it before we release that major version.
+
+Pre-releases are ideal for short term investigation or early access to the community when a breaking change might include something difficult to implement. They’re also generally easier to manage compared to feature flags. Consider if you could use a pre-release instead of a feature flag.
```

Comment:
```suggestion
Pre-releases are ideal for short term investigations or providing early access to the community so they have time to assess a breaking change that might be difficult to implement. Also, pre-releases are generally easier to manage compared to feature flags. Consider if you could use a pre-release instead of a feature flag.
```